test/e2e/ethereum_contract_test.go

```diff
@@ -161,7 +162,7 @@ type EthereumContractTestSuite struct {
 	suite.Suite
 	testState       *testState
 	contractAddress string
-	interfaceID     string
+	contractID      string
```

Comment:
Why rename this back to contractID? Not a big deal, but it is the ID of an FFI / Contract Interface.

Reply:
This was probably an unintended change while merging/rebasing branches. I will change it back.